java/vector/src/main/java/org/apache/arrow/vector/util/DateUtility.java

```diff
@@ -620,10 +618,10 @@
     }
   }
 
-  public static final DateTimeFormatter formatDate = DateTimeFormat.forPattern("yyyy-MM-dd");
-  public static final DateTimeFormatter formatTimeStampMilli = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss.SSS");
-  public static final DateTimeFormatter formatTimeStampTZ = DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss.SSS ZZZ");
-  public static final DateTimeFormatter formatTime = DateTimeFormat.forPattern("HH:mm:ss.SSS");
+  public static final DateTimeFormatter formatDate = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
Looks like it will probably not affect you here, but there are some subtle differences between joda time format patterns and java time format patterns. The main one that has caused me trouble is the meaning of `Y` is different between the two.

Reply:
Thanks @elahrvivaz !  I'll make a note of that in the description